Gallery/GalleryCollectionViewController.swift

```diff
@@ -13,8 +13,6 @@ class GalleryCollectionViewController: UICollectionViewController {
     
     private let sectionInsects: UIEdgeInsets = UIEdgeInsets(top: 50, left: 10, bottom: 50, right: 10)
     private let itemsPerRow: CGFloat = 2
-    private var images = ImageStorage.shared.images
-    private var indexPath = ImageStorage.shared.indexPath
```

Comment:
You deleted indexPath property from controllers. Look like you dont use ImageStorage.shared.indexPath property in you project anymore. Could you please check it?

Reply:
Also, please remove commented string 
//view.layer.addSublayer(gradientLayer)
from DetailPaggedViewController

Reply:
I removed comment.
I still need ImageStorage.shared.indexPath property because i need it for scrolling in DetailPaggedViewController

Reply:
removed unnecessary indexPath